src/lib/components/legend/Legend.svelte

```diff
@@ -42,167 +33,96 @@
 		</svg>
-		<p class="font-['Basis_Grotesque'] text-xs leading-tight text-gray-500">
-			Click categories below to filter the map
-		</p>
+		<p class="font-basis text-xs leading-tight">Click categories below to filter the map</p>
 	</div>
-	<div class="mode-selector relative mb-2 grid grid-cols-3">
+	<div
+		class="relative mb-2 grid grid-cols-3 gap-x-px border border-solid border-gray-200 bg-gray-200"
+	>
 		<div
-			class="mode-selector__background"
-			class:agency={$visualState.colorMode === 'agency'}
-			class:category={$visualState.colorMode === 'category'}
-			class:funding={$visualState.colorMode === 'fundingSource'}
+			class={[
+				'absolute left-0 top-0 z-10 h-full w-1/3 bg-earth transition-transform duration-300',
+				{
+					'translate-x-0': $visualState.mode === 'agency',
+					'translate-x-full': $visualState.mode === 'category',
+					'translate-x-[200%]': $visualState.mode === 'fundingSource'
+				}
+			]}
 		></div>
 		{#each modes as mode}
-			<div class="mode-selector__radio-container relative">
+			<div class="relative bg-white">
 				<input
 					type="radio"
-					bind:group={$visualState.colorMode}
+					bind:group={$visualState.mode}
 					id="{mode.value}-radio"
 					value={mode.value}
-					class="mode-selector__radio-input absolute opacity-0"
-					on:change={handleModeChange}
+					class="absolute opacity-0"
+					onchange={handleModeChange}
 				/>
 				<label
 					for="{mode.value}-radio"
-					class="mode-selector__radio-label relative z-10 block cursor-pointer py-1.5 text-center font-['PolySans'] text-xs"
-					class:active={$visualState.colorMode === mode.value}
+					class={[
+						'relative z-10 block cursor-pointer py-1.5 text-center font-poly text-xs transition-colors',
+						$visualState.mode === mode.value ? 'text-white' : 'hover:bg-gray-100'
+					]}
 				>
 					{mode.label}
 				</label>
 			</div>
 		{/each}
 	</div>
 	<div class="space-y-0.5">
-		{#each legendItems[$visualState.colorMode] as item}
+		{#each legendItems[$visualState.mode] as item}
 			<button
-				class="flex w-full items-center gap-2 rounded border border-gray-200 bg-slate-50/90 px-1.5 py-0.5 transition-colors hover:bg-gray-100"
-				class:opacity-40={$visualState.filters.size > 0 && !isItemActive(item.label)}
-				on:click={() => toggleFilter(item.label)}
+				class={[
+					'flex w-full items-center gap-2 rounded border border-gray-200 bg-slate-50/90 px-1.5 py-0.5 transition-colors hover:bg-gray-100',
+					{ 'opacity-40': $visualState.filters.size > 0 && !$visualState.filters.has(item.label) }
+				]}
+				onclick={() => toggleFilter(item.label)}
 			>
-				<div class="h-2 w-2 rounded-full" style="background-color: {item.color}"></div>
-				<span class="text-left font-['Basis_Grotesque'] text-xs">{item.label}</span>
+				<span class="h-2 w-2 rounded-full" style="background-color: {item.color}"></span>
+				<span class="font-basis text-xs">{item.label}</span>
 			</button>
 		{/each}
 	</div>
 </div>
-
-<style lang="postcss">
```

Comment:
I shifted all of our styles out of here and into Tailwind classes. I know it's weird, but the primary use case for Tailwind is that you can see _all of your styles directly in the HTML markup_. Having to shift between the markup and styles defined down here just adds that little extra bit of cognitive load. It's also a little funky to go "halfway" on embracing Tailwind; I'm going to encourage is to take the full plunge.